astropy/io/fits/scripts/fitsdiff.py

```diff
@@ -319,3 +319,7 @@ def main():
     finally:
         if close_file:
             out_file.close()
+        for handler in log.handlers:
```

Comment:
Could you include a comment why this is necessary?

Reply:
Yep.